### PR TITLE
ci: add data service deployment scripts

### PR DESCRIPTION
**Pull request by dgauldie**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Resolves #(issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide reproduction instructions and list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: kubernetes/local/data-service-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            - name: SQL_URL
+              value: data-service-postgres
+            - name: SQL_PORT
+              value: "5020"
+            - name: SQL_USER
+              value: dev
+            - name: SQL_PASSWORD
+              value: dev
+            - name: DKG_URL
+              value: http://34.230.33.149
```
Comment (chris-dickson) [inline kubernetes/local/data-service-deployment.yaml:35]: What is this?

**Review by chris-dickson — COMMENTED** [review]
Does the GraphDB need an associated volume for persistence?  Is it only used an intermediate database?  eg, data is staged, some things are computed and it's deleted?

File: kubernetes/local/deploy-terarium.sh [hunk 2]
```diff
@@ -7,7 +7,7 @@ function start_gateway() {
 function start_hmi-server() {
     # Wait for Keyclock, the hmi-server test its existence on start
     kubectl rollout status --filename 'gateway-keycloak-*.yaml'
-    kubectl apply --filename 'hmi-server-*.yaml','mock-data-service-*.yaml'
+    kubectl apply --filename 'hmi-server-*.yaml','data-service-*.yaml'
```
Comment (YohannParis) [inline kubernetes/local/deploy-terarium.sh:10]: This is fine for now, but should be put on it's separate function waiting for ES, PostGres, Neo4J and S3-like storage to be up and running.

File: kubernetes/local/data-service-deployment.yaml (hunk 1 above)
Comment (YohannParis) [reply kubernetes/local/data-service-deployment.yaml:35]: This is from HMS to access their **D**omain **K**nowledge **G**raph service.

Comment (YohannParis) [reply kubernetes/local/data-service-deployment.yaml:35]: This is going to change with the forking of [MIRA](https://github.com/DARPA-ASKEM/mira) on our organization.

**Review by YohannParis — APPROVED** [review]
LGTM, credential is what I was looking into!

**Review by YohannParis — APPROVED** [review]

**Review by YohannParis — APPROVED** [review]

**Pull request merged**